app/Http/Controllers/CourseController.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Course;
+use App\Models\Tag;
+use App\Models\User;
+use Illuminate\Http\Request;
+
+class CourseController extends Controller
+{
+    public function index()
+    {
+        $courses = Course::orderBy('id')->paginate(config('constants.pagination'));
+        $mentors = User::mentor()->get();
```

Comment:
dòng này là lấy toàn bộ mentor từ hệ thống ra à em ? khác gì `Mentor::all()` nhỉ ?

Reply:
Theo em hiểu là all() là phương thức tĩnh trong Eloquent\Model và không thể có sự thay đổi nào. Còn get() là phương thức của Eloquent\Builder và khi có truy vấn thì e phải dùng get(). mentor() ở đây là 1 scope để e truy vấn đến những user có role = mentor. Nếu mentor là 1 model thì em sẽ dùng Mentor::all() ạ, trường hợp này thì phải dùng User::mentor()->get();

Reply:
@Lamnq99 ok em, nhìn khá giống với quan hệ mentor() nên anh có thắc mắc như vậy, hoá ra chỗ này là queryScope